vclib-openid/src/commonTest/kotlin/at/asitplus/wallet/lib/oidc/OidcSiopInteropTest.kt

```diff
@@ -47,126 +48,108 @@ class OidcSiopInteropTest : FreeSpec({
+        aWRlbnRpdHkiLCJmb3JtYXQiOnsibXNvX21kb2MiOnsiYWxnIjpbIkVTMjU2IiwiRVMzODQiLCJFUzUxMiIsIkVkRFNB
+        IiwiRVNCMjU2IiwiRVNCMzIwIiwiRVNCMzg0IiwiRVNCNTEyIl19fSwiY29uc3RyYWludHMiOnsiZmllbGRzIjpbeyJw
+        YXRoIjpbIiRbJ2V1LmV1cm9wYS5lYy5ldWRpdy5waWQuMSddWydmYW1pbHlfbmFtZSddIl0sImludGVudF90b19yZXRh
+        aW4iOmZhbHNlfSx7InBhdGgiOlsiJFsnZXUuZXVyb3BhLmVjLmV1ZGl3LnBpZC4xJ11bJ2JpcnRoX2RhdGUnXSJdLCJp
+        bnRlbnRfdG9fcmV0YWluIjpmYWxzZX0seyJwYXRoIjpbIiRbJ2V1LmV1cm9wYS5lYy5ldWRpdy5waWQuMSddWydnaXZl
+        bl9uYW1lJ10iXSwiaW50ZW50X3RvX3JldGFpbiI6ZmFsc2V9XX19XX0sInN0YXRlIjoibGlmLVAwMldtMjV0aFRLb2NS
+        ZUVqUWFyLUtxbW1BWU1vN3hXX25OcVRtdW02eXEwbF8xcXFMSXhuMkJZVndLRFBVX2RkMEJHWmpOMUNnYTRrVk9fbnci
+        LCJpYXQiOjE3MTUxNTAwNzMsImNsaWVudF9tZXRhZGF0YSI6eyJhdXRob3JpemF0aW9uX2VuY3J5cHRlZF9yZXNwb25z
+        ZV9hbGciOiJFQ0RILUVTIiwiYXV0aG9yaXphdGlvbl9lbmNyeXB0ZWRfcmVzcG9uc2VfZW5jIjoiQTEyOENCQy1IUzI1
+        NiIsImlkX3Rva2VuX2VuY3J5cHRlZF9yZXNwb25zZV9hbGciOiJSU0EtT0FFUC0yNTYiLCJpZF90b2tlbl9lbmNyeXB0
+        ZWRfcmVzcG9uc2VfZW5jIjoiQTEyOENCQy1IUzI1NiIsImp3a3NfdXJpIjoiaHR0cHM6Ly92ZXJpZmllci1iYWNrZW5k
+        LmV1ZGl3LmRldi93YWxsZXQvamFybS9saWYtUDAyV20yNXRoVEtvY1JlRWpRYXItS3FtbUFZTW83eFdfbk5xVG11bTZ5
+        cTBsXzFxcUxJeG4yQllWd0tEUFVfZGQwQkdaak4xQ2dhNGtWT19udy9qd2tzLmpzb24iLCJzdWJqZWN0X3N5bnRheF90
+        eXBlc19zdXBwb3J0ZWQiOlsidXJuOmlldGY6cGFyYW1zOm9hdXRoOmp3ay10aHVtYnByaW50Il0sImlkX3Rva2VuX3Np
+        Z25lZF9yZXNwb25zZV9hbGciOiJSUzI1NiJ9fQ.-qNymEtka8jQvM3F2Rjl551kHrvnZTknfy3gD4K9o3QUKWlYQzeie
+        mBMp1YvSfOzMf-U-sryEJV8T9ANWiQNAw
         """.trimIndent()
 
         val jwkset = """
             {
                 "keys": [
                     {
-                        "alg": "ECDH-ES",
-                        "crv": "P-256",
-                        "kid": "1835c633-bd3f-429e-8dfa-64596b83aa0c",
                         "kty": "EC",
                         "use": "enc",
-                        "x": "lBeONku60ShqCvndUdFVubOCCuvMjWTmElaxgHWbuMo",
-                        "y": "NHYLE--QpTqc9vGrTLoq1dm2c86AC6af6xiHiLpKjdk"
+                        "crv": "P-256",
+                        "kid": "0e30be2d-1e8f-482d-b345-26f9f06b4243",
+                        "x": "xFWlKn9MeGVkvtQgbVIqC0Qc6499LN9eEGixzYsJ3tg",
+                        "y": "IcS_SK-kAeb4xaDM8qMlunPf5_LjSgkZ_xPj4kutVKs",
+                        "alg": "ECDH-ES"
                     }
                 ]
             }
-
         """.trimIndent()
 
         holderSiop = OidcSiopWallet.newInstance(
             holder = holderAgent,
             cryptoService = holderCryptoService,
             remoteResourceRetriever = {
                 if (it == "https://verifier-backend.eudiw.dev/wallet/jarm/" +
-                    "WLFJEn9AGbJfAcEyaQTzzxueqmeRazmsHIkxMRTkGRL1zyI7un-KJWaXtulrfiSS38LlU5ABDB9Zdsfq_11r8Q/jwks.json"
+                    "lif-P02Wm25thTKocReEjQar-KqmmAYMo7xW_nNqTmum6yq0l_1qqLIxn2BYVwKDPU_dd0BGZjN1Cga4kVO_nw/jwks.json"
```

Comment:
Why did this change?

Reply:
I just did a new presentation and copied the new parameters there

It seems like their presentation request changed from back then to now, so I figured it would be a good idea to update this